apps/backoffice-v2/src/lib/blocks/variants/PDFRevisionBlocks/hooks/usePDFRevisionBlocks/usePDFRevisionBlocks.tsx

```diff
@@ -0,0 +1,42 @@
+import { useFilterId } from '@/common/hooks/useFilterId/useFilterId';
+import { useWorkflowByIdQuery } from '@/domains/workflows/hooks/queries/useWorkflowByIdQuery/useWorkflowByIdQuery';
+import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import { useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+
+export const usePDFRevisionBlocks = () => {
+  const { entityId: workflowId } = useParams();
+  const filterId = useFilterId();
+  const { data: workflow } = useWorkflowByIdQuery({
+    workflowId: workflowId as string,
+    filterId: filterId as string,
+  });
+
+  const blocks = useMemo(() => {
+    return createBlocksTyped()
+      .addBlock()
+      .addCell({
+        type: 'container',
```

Comment:
Make sure with Nitzan if the intention wasn't using `type: block` that has the border and so on of the other blocks in the backoffice.